### ISSUE TITLE
Akka Error and OCSP verification failure 

### ISSUE THREAD

**Issue by egobsv** [nordic-institute/X-Road#893]
Hi, 
My security server (version 6.22) is failing with this error:
"Server.ClientProxy.SslAuthenticationFailed.InvalidCertPath.IncorrectValidationInfo","message":"OCSP response is too old (thisUpdate: Wed Feb 24 14:36:48 CST 2021)"

I already checked the time on my OCSP responders and confirmed that it is in sync with that of the security server. I also tried clearing  the OCSP cache at /var/cache/xroad/*ocsp. And disabled the verifyNextUpdate parameter in the central server. 

Looking at my  signer.log I keep get this error after a fresh restart:
INFO  [Signer-akka.actor.default-dispatcher-3] ee.ria.xroad.signer.OcspClientJob - OCSP-response refresh cycle successfully completed, continuing with normal scheduling
ERROR [Signer-akka.actor.default-dispatcher-15] akka.remote.EndpointWriter - AssociationError [akka.tcp://Signer@127.0.0.1:5558] <- [akka.tcp://SignerConsole@127.0.0.1:33297]: Error [Shut down address: akka.tcp://SignerConsole@127.0.0.1:33297]

And then when any query is attempted, the following message is  added to signer.log
WARN  [Signer-akka.actor.default-dispatcher-47] e.r.x.s.c.OcspResponseManager - Could not find certificate for hash 1d4a0316383......
WARN  [Signer-akka.actor.default-dispatcher-47] e.r.x.s.c.OcspResponseManager - Could not acquire an OCSP response for certificate 1d4a0316383......

I checked the security server's diagnostic page, certificates are displayed as good and OCSP and TSA services have a green icon. What could be happening?

Thank you for time and help!
Eric.

**Comment by petkivim** [comment]
Hi @egobsv 

Have there been any changes regarding the approved Certificate Authorities on the Central Server? Removing the CA that issued the certificate from the configuration could explain the issue.

It seems to be the authentication key that is failing. Have you checked the "keys" page to see what is the status of the authentication key & certificate?

The following error can be ignored. It's a false error and the issue has been fixed in version 6.24.0. More information can be found [here](https://jira.niis.org/browse/XRDDEV-63).

```
ERROR [Signer-akka.actor.default-dispatcher-15] akka.remote.EndpointWriter - AssociationError [akka.tcp://Signer@127.0.0.1:5558] <- [akka.tcp://SignerConsole@127.0.0.1:33297]: Error [Shut down address: akka.tcp://SignerConsole@127.0.0.1:33297]
```

Could you change the signer logging level to `debug` and provide the full signer log? That would help us to narrow the cause of the problem. Instructions on how to change the logging level can be found [here](https://confluence.niis.org/pages/viewpage.action?pageId=6783404).

Best regards,
Petteri

**Comment by egobsv** [comment]
Hi Petteri, thanks for your reply. After a few more hours of investigation the source of the problem turned out to be a change they recently applied on our network firewalls.

Thanks again for your help.
